README.md

```diff
@@ -83,6 +84,9 @@ const IconfontWebpackPlugin = require('iconfont-webpack-plugin');
                   resolve: loader.resolve,
                   fontNamePrefix: 'custom-',
                   modules: false,
+                  compilerOptions: {
```

Comment:
Imho the name compilerOptions doesn't explain what it is doing.  
Do you believe we need more options than the fontHeight?

Would it make sense to add fontHeight directly to the options and provide a default?

Reply:
I guess that is for you to decide? Would you prefer just having the fontHeight by default? It just means that if there are others who want to adjust the svg options then it would require another modification down the track.

Reply:
Cool - so if you don't see any downside (beside that we would have to add other svg options as well once they are needed) I would definitely prefer adding only the height.

Reply:
Ok, that makes my PR super simple. I will update it tomorrow (late in Australia now).

Reply:
Perfect - I'll try to release the package directly tomorrow morning 👍